Materials2050_Adapter/Materials2050Adapter.cs

```diff
@@ -37,34 +41,23 @@ public partial class Materials2050Adapter : BHoMAdapter
         /**** Constructors                              ****/
         /***************************************************/
 
-        [Description("Adapter for Materials2050.")]
-        [Output("The created Materials2050 adapter.")]
-        public Materials2050Adapter()
+        [Description("Adapter to connect to 2050 Materials API.")]
+        [Input("apiToken", "Provide 2050 Materials API Token")]
+        [Output("adapter", "Adapter results")]
+        public Materials2050Adapter(string apiToken = "", bool active = false)
```

Comment:
I would look long term to move `apiToken` to a config object and remove `bool active` - @Tom-Kingstone can provide some help as he's done a lot with me on this for environment toolkits recently - I can also help when I get back from holiday but if you want quicker support, speak to @Tom-Kingstone 😄 

Obviously - not needed for prototype, happy to leave as is for now - just suggesting for later improvements.

Reply:
Good shout 👍